benchmarks/200.multimedia/220.video-processing-gpu/python/function.py

```
@@ -0,0 +1,200 @@
+
+    with torch.no_grad():
+        for _ in range(num_iters):
+            video = torch.relu(conv(video))
+
+    # Back to uint8 on CPU: (T, H, W, C)
+    video = (video.clamp(0.0, 1.0) * 255.0).byte()
+    video_np_out = video.permute(0, 2, 3, 1).cpu().numpy()
+
+    # Encode processed video with OpenCV (CPU)
+    base = os.path.splitext(os.path.basename(video_path))[0]
+    out_path = f"/tmp/processed-{base}.mp4"
```

Ruff (0.14.8)

Probable insecure usage of temporary file or directory: "/tmp/processed-"

(S108)